src/plugins/spherical/spherical.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Spherical Positioning Plugin
+ * ============================
+ *
+ * This plugin provides support for positioning steps with polar or shperical coordinates.
+ * This is often more convenient when arranging slides along a circle or similar. It also
+ * supports relative positioning.
+ *
+ * Example:
+ *
+ *  <!-- Position step 1000 px distance to origin at an angle of 50 degree. -->
+ *  <div class="step" data-spherical-dist="1000" data-spherical-rho="50">
+ *
+ *  <!-- Position step 500 px distance to origin at a primary angle of 270 degree and a
+ *       secondary angle of -30 degree (= 330 degree) -->
+ *  <div class="step" data-spherical-dist="500" data-spherical-rho="270" data-spherical-theta="-30">
+ *
+ *
+ * Following html attributes are supported for step elements:
+ *
+ *     data-spherical-distance
+ *     data-spherical-rho
+ *     data-spherical-theta
+ *
+ *     data-spherical-rel-distance
+ *     data-sphercial-rel-rho
+ *     data-spherical-rel-theta
+ *
+ * Non-zero relative values are inherited from the previous step. This makes it easy to
+ * create a presentation where subsequent slides are arranged along a circle.
```

Comment:
> I now realize data-rel-to itself isn't inherited. It would probably make sense?

Btw... I implemented this, and don't like it. (For rel.js I mean.) In this case it's better and much less confusing that data-rel-to must always be explicitly present. So what you implemented in rel.js is good.